playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -1,3 +1,13 @@
+- name: Configure ntp
+  hosts:
+    - oo_nodes_to_config
+    - oo_etcd_to_config
+  vars:
+    ntp_require_clock_sync: "{{ openshift_require_clock_sync | default(false) | bool }}"
+  roles:
+    - role: ntp
+      when: not openshift.common.is_containerized | bool and openshift_manage_clock_method is defined and
```

Comment:
I don't think we'd want to skip ntp configuration on containerized hosts (though we may want to fail if is_containerized is true and is_atomic is true and openshift_manage_clock_method isn't available).